### DISCUSSION TITLE
Annotation column

### DISCUSSION THREAD

**Discussion by saadzmirza** [sumatrapdfreader/sumatrapdf#3974] [Ideas]
Acrobat has an annotation column view where I can easily review all the comments on my document and add more![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/130395874/d6c45f8e-d0bb-4c8f-ba31-2fde377434b8)

**Comment by kjk** [comment]
In latest Sumatra use right-click (context) menu and `Edit Annotations". See https://www.sumatrapdfreader.org/docs/Editing-annotations

**Reply by Sjotroll** [reply]
Hi, the annotations tab in Sumatra lets you see the comment in only one annotation at a time. So if I'm searching for something I'd have to look through each annotation, which is obviously impractical. It would be a great addition that would make me leave Acrobat completely to have a tab that show all the comments for all annotations, just like in Acrobat, as saadzmirza shown. If you could also add the color of the annotation to be visible in the tab, that would be great!
